lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,38 @@
+local CATEGORY_NAME = "Fun"
+
+local MIN_STRENGTH = 6000	
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+	for _, v in pairs( targets ) do 
+		
+	if not v:IsPlayer() then return end
+	if not v:Alive() then return end
+
+	if v:IsFrozen() then
+		
+	v:Freeze( false )
+					
+	end
+	
+	if v:GetMoveType() == MOVETYPE_NOCLIP then
+			
+	v:SetMoveType( MOVETYPE_WALK )
+			
+	end
```

Comment:
Is there ever a situation where you wouldn't want them to be in `MOVETYPE_WALK`? If not, I would suggest one of two things:

1. Simply:
```lua
ply:SetMoveType( MOVETYPE_WALK )
```

2.
```lua
if ply:GetMoveType() ~= MOVETYPE_WALK then
    ply:SetMoveType( MOVETYPE_WALK )
end
```